Makefile

```
@@ -8,20 +8,26 @@ LDFLAGS := -X '$(GO_PKG_NAME)/pkg/version.CommitFromGit=$(GIT_COMMIT)' \
            -X '$(GO_PKG_NAME)/pkg/version.BuildDate=$(BUILD_DATE)' \
            -X '$(GO_PKG_NAME)/pkg/version.GitTreeState=$(GIT_TREE_STATE)'
 
-.PHONY: verify test lint clean unit integration example-tests framework-tests
+.PHONY: verify test lint clean unit integration example-tests example-noisy-tests framework-tests
 
```

checkmake (0.3.2)

[warning] Required target "all" must be declared PHONY.

(minphony)